Abilities/npc_dota_hero_skywrath_mage/skywrath_mage_concussive_shot.cs

```diff
@@ -39,5 +44,58 @@ protected override float RawDamage
                 return this.Ability.GetAbilitySpecialData("damage");
             }
         }
+
+        public Hero TargetHit
+        {
+            get
+            {
+                return EntityManager<Hero>.Entities.OrderBy(x => 
```

Comment:
Maybe you should first filter by possible targets, and then order -> should be more efficient

Reply:
okey